src/lib.rs

```diff
@@ -11,4 +13,32 @@ mod tests {
         let result = add(2, 2);
         assert_eq!(result, 4);
     }
+
+    #[test]
+    fn fen_starting_board() {
```

Comment:
I would probably add the FEN tests to the FEN module, so they're easier to find.

Reply:
good idea!